api/admin/opds.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, circulation, library, test_mode=False):
     def annotate_work_entry(self, work, active_license_pool, edition, identifier, feed, entry):
 
         super(AdminAnnotator, self).annotate_work_entry(work, active_license_pool, edition, identifier, feed, entry)
-        VerboseAnnotator.annotate_work_entry(work, active_license_pool, edition, identifier, feed, entry)
+        VerboseAnnotator.add_ratings(work, entry)
```

Comment:
This is still a little inelegant, but it avoids doing the work of `annotate_work_entry` twice by only doing the part of it that is specific to VerboseAnnotator. There may be a way to make multiple inheritance do this job but I've never trusted multiple inheritance in Python.